ts/utils/crypto.ts

```diff
@@ -52,6 +56,15 @@ export const checkPublicKeyExists = (keyId: string) =>
     TE.getOrElse(() => T.of(false))
   )();
 
+export const taskRegenerateKey = (keyId: string) =>
```

Comment:
Is there possibly be an inconsistency on the input name of the parameter? Here and someplace else we are using "keyId" while there and someplace else we are using "keyTag". Do they refer to the same entity? If so, shall we use the same name?